### ISSUE TITLE
atmospherejs.com always shows README from github master

### ISSUE THREAD

**Issue by udondan** [percolatestudio/atmosphere#163]
It seems like atmospherejs.com always fetches the most recent README from github master.

I see how this is convenient as I do not have to release a new version to fine-tune the description, killing typos and the like.

But this could be a problem, as the master README might not be the same as the version from the published tag and the description might differ from how the released version actually works.

**Comment by tmeasday** [comment]
It's a good point @udondan - although I think it's tricky to get right in general..

**Comment by udondan** [comment]
Can't you just fetch the content from the correct tag? e.g. `https://github.com/udondan/meteor-contact/blob/v0.2.0/README.md` instead of `https://github.com/udondan/meteor-contact/blob/master/README.md`. You just need to replace master with the tag which you know since you show the version number on the page :)

**Comment by tmeasday** [comment]
Right. But it might not be what a package author expects -- they have to publish a new version of the package to update the readme? 

I think it's tricky. I think in general it's best if people use master for the latest version of the package and   do any further development on another branch to avoid this problem. But it's tricky to enforce that too..

**Comment by udondan** [comment]
> they have to publish a new version of the package to update the readme?

Well, as written, I see how it is very convenient the way it is. But the readme is part of the package, so no matter where the user develops (branch, master) I think it should be displayed the readme which belongs to the package. Even if that is not very handy, it IMHO is the correct way.

npm is doing it like that, showing the readme of the release. (not that npm is a good example for something ;))

But with the same argument `mrt` could always install the master of a package instead of the tag referenced in the smart.json. :)

But that's only my point of view. I don't want to force a change. Just mentioned it when noticed. When this is by design, that's totally OK too. :)

**Comment by tmeasday** [comment]
It's a good point about publishing going off master too but that'd probably be a bit extreme :)

I think it's a really good point that you make and honestly I don't know what the best answer is. I think it's something worth revisiting post-0.9.0 when MDG are hosting the packages and when I don't think they plan on storing a git URL for packages. At that point, it'll be more of atmosphere's responsibility to sort this stuff out.

**Comment by udondan** [comment]
Closing to get this off of my dashboard.
